src/js_stream.cc

```diff
@@ -115,7 +115,7 @@ void JSStream::New(const FunctionCallbackInfo<Value>& args) {
   // normal function.
   CHECK(args.IsConstructCall());
   Environment* env = Environment::GetCurrent(args);
-  JSStream* wrap;
+  JSStream* wrap = NULL;
```

Comment:
This isn't necessary, either the variable will already be assigned below or the process aborts because of an unreachable condition.